bundles/org.openhab.binding.ojelectronics/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -22,10 +22,10 @@
 			</parameter>
 			<parameter name="apiUrl" type="text" required="true">
 				<label>API-URL</label>
-				<description>URL to cloud API-service.</description>
+				<description>URL to cloud API-service and Socket-Notification.</description>
```

Comment:
Please update I18N properties after this change. For reference: https://www.openhab.org/docs/developer/utils/i18n.html#generating-i18n-properties-file

Reply:
Done